packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text from '../Text';
+
+function Term({
+  label,
+  after,
+  endAlign,
+  keyTextRegular,
+  children,
+  cx,
+  styles,
+}: {
+  label: string;
+  after?: NonNullable<React.ReactNode>;
+  endAlign?: boolean;
+  inline?: boolean;
+  keyTextRegular?: boolean;
+  children: NonNullable<React.ReactNode>;
+} & WithStylesProps) {
+  return (
+    <div className={cx(styles.container)}>
+      <Row inline={!endAlign} after={endAlign && after}>
+        <dt>
+          <Text inline small={!keyTextRegular}>
+            {label}
+          </Text>
+        </dt>
+        {!endAlign && (
+          <Spacing inline left={1}>
+            {after || ''}
+          </Spacing>
+        )}
+      </Row>
+      <dd>
+        <Text>{children}</Text>
+      </dd>
+    </div>
+  );
+}
+
+export default withStyles(() => ({
+  container: {
+    '@selectors': {
+      '> dd': {
```

Comment:
you can do the reset on the `dd` itself

Reply:
Sorry I don't totally get this, you mean something like
```
dd: {
    marginInlineStart: 0,
  },
```
?

I haven't been able to get that to work